backend/src/v5/utils/permissions/permissions.js

```diff
@@ -35,23 +35,15 @@ Permissions.isTeamspaceAdmin = async (teamspace, username) => {
 Permissions.hasAccessToTeamspace = hasAccessToTeamspace;
 
 Permissions.isProjectAdmin = async (teamspace, project, username) => {
-	const admins = await getProjectAdmins(teamspace, project);
-	return admins.includes(username);
+	const isTsAdmin = await Permissions.isTeamspaceAdmin(teamspace, username);
```

Comment:
don't do this - it's separated on purpose.

Call both isTeamspaceAdmin and isProjectAdmin if you need to check both.